Survey/Survey/Models/SurveyModel.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Combine
+
+class SurveyViewModel: ObservableObject {
+    @Published var inquiries: [Inquiry] = []
+    private var cancellables = Set<AnyCancellable>()
+    private let mockServer = MockServer()
+
+    init() {
+        loadInquiries()
+    }
+
+    func loadInquiries() {
+        attemptLoadInquiries()
+    }
+
+    private func attemptLoadInquiries() {
+        mockServer.loadInquiries()
```

Comment:
this doesn't seem right -- basically you're hardcoding a mock server into the app. Instead, what you might want to do is use a Proxy design pattern to hide the exact implementation of your serverside behind an interface, then have a mock and a real implementation of that interface. That way switching to the real one will be very simple. Right now it's going to require removing this mockserver, right?

Reply:
Got it. Will work on it.

Reply:
I'm not seeing an interface-driven implementation here. Just mockServer init has moved from this function to the class.

Reply:
This will be changed

Reply:
Changing in my PR.